src/tex.js

```diff
@@ -203,8 +231,26 @@ MathJax.Hub.Register.StartupHook('TeX Jax Ready', function () {
       return token;
     },
     arabicNumber: (function () {
-      var englishNumbersRegExp = /[0-9]/g;
-      var numbersMap = MathJax.Hub.config.Arabic.numbersMap;
+      // check for MathJax options
+      useHindiNumeral = MathJax.Hub.config.useHindiNumeral
+      useArabicDecimalSeparator = MathJax.Hub.config.useArabicDecimalSeparator
+      
+      // Default options
+      if (useHindiNumeral == null) useHindiNumeral = true
+      if (useArabicDecimalSeparator == null) useArabicDecimalSeparator = true
```

Comment:
I suggest that we identify a better pattern than this e.g. use falsey variables like: `disableArabicDecimalSeparator` and `disableHinduArabicNumbers`